### PR TITLE
Added copy & rename functionality to pricing rules panel.

### PR DESCRIPTION
**Pull request by AndreasKoettl95**
## Changes in this pull request  
Pricing rules panel now includes copy & rename feature in context menu of a pricing rule.
This should speed up the process of making new rules and managing existing ones.
The behaviour is similar to copying & renaming items in the Documents/Assets/DataObjects trees.

### CODE REVIEW

**Comment by fashxp** [comment]
Hi, thx very much for your PR. Can you please sign our CLA so that we can merge?

File: bundles/EcommerceFrameworkBundle/Controller/PricingController.php [hunk 1]
```diff
@@ -191,6 +192,108 @@ public function deleteAction(Request $request)
         return $this->adminJson($return);
     }
 
+    /**
+     * @Route("/copy", methods={"POST"})
```
Comment (NiklasBr) [inline bundles/EcommerceFrameworkBundle/Controller/PricingController.php:196]: Should this implement JS Routing (#6146)?

Comment (dpfaffenbauer) [reply bundles/EcommerceFrameworkBundle/Controller/PricingController.php:196]: not yet implemented in ecommerce framework

**Comment by brusch** [comment]
Closed in favor of #7127 with JS Routing and rebased onto latest master (Travis).

**Pull request closed without merging**